utilities/rocAL/rocAL_dataloader/rocAL_dataloader.cpp

```diff
@@ -165,7 +165,7 @@ int main(int argc, const char ** argv)
      RocalImage image0;
      image0 = input1;
     // just do one augmentation to test
-    // image1 = rocalRain(handle, image0, true);
+    rocalRain(handle, image0, true);
```

Comment:
@swetha097 Was this change meant to be in this PR?

Reply:
Warnings were also resolved in addition the CMakeLists changes